package.json

```diff
@@ -42,5 +42,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```

Comment:
@bismitaguha I think @sidvenu asked you to shift this to another pr. He has created an issue also for the same. Pls check